src/repo/mem.rs

```diff
@@ -2,11 +2,11 @@
 use crate::error::Error;
 use crate::repo::{BlockStore, Column, DataStore};
 use async_std::path::PathBuf;
+use async_std::sync::{Arc, Mutex};
```

Comment:
Since I went and took a look:

 * Arc is just re-exported from `std::sync`
 * `async_std::sync::Mutex` vs. `futures::sync::Mutex` the former uses a private abstraction `async_std::sync::WakerSet` which looks somewhat similar than what `futures::sync::Mutex` uses internally (`Slab<Waker>` or something along those lines) but `WakerSet` is likely used in many different scenarios

Shame they haven't made it public, it would be that `Vec<Waker>` used inside `Subscription`.